<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Product</class>
 <widget class="QMainWindow" name="Product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>13</y>
      <width>221</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addProduct">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить продукт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteLProduct">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить продукт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sellProduct">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Продать продукт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Information">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Информация о продукте</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculateTotalBudget">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Общий бюджет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sale">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Продано</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Remains">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Остаток</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revizion">
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ревизия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>Название</string>
        </property>
        <property name="font">
         <font>
          <family>Century</family>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Артикул</string>
        </property>
        <property name="font">
         <font>
          <family>Century</family>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Цена</string>
        </property>
        <property name="font">
         <font>
          <family>Century</family>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Количество</string>
        </property>
        <property name="font">
         <font>
          <family>Century</family>
         </font>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Название: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Код:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="skuLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Количество:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="quantitySpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Цена:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="priceSpinBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
